src/sass/partials/_hero.scss

```diff
@@ -56,17 +59,147 @@
 .hero__lady_with_icecream {
   display: none;
 }
+.hero__arrow {
+  display: none;
+}
+//tabet
+@media (min-width: 768px) {
+  .hero-container {
+    background-color: $IBgColor;
+
+    background-image: url(../images/ellipse.png), url(../images/ellipse.png);
+    background-repeat: no-repeat, no-repeat;
+    background-position: 730px 9px, 730px 75px;
+    background-size: 16px 16px, 16px 16px;
+    position: relative;
+  }
+  .hero-container::before {
+    background-image: url(../images/ice-cream_tablet.png);
+    position: absolute;
+    background-repeat: no-repeat;
+    height: 336px;
+    width: 100vw;
+    top: -23px;
+    left: 378px;
+    content: '';
+    z-index: 3;
+  }
+  .hero-container:after {
+    background-image: url(../images/background-circle_tablet.png);
+    position: absolute;
+    background-size: 324px 322px;
+    background-repeat: no-repeat;
+    height: 100%;
+    width: 100vw;
+    top: -42px;
+    left: 307px;
+    content: '';
+    z-index: 2;
+  }
+  .hero__slogan-buttons {
+    padding-top: 15px;
+    padding-bottom: 66px;
+    padding-left: 14px;
+  }
+  .hero__slogan {
+    text-transform: uppercase;
+    height: 80px;
+    width: 168px;
+    font-family: $fontSecondary;
+    font-size: 22px;
+    font-weight: 400;
+    line-height: 26px;
+    letter-spacing: 0.04em;
+    text-align: left;
+    color: $IColor;
+    margin-bottom: 20px;
+    margin-right: 495px;
+  }
+  .hero__slogan-span {
+    font-family: $fontSecondary;
+    font-size: 34px;
+    font-weight: 400;
+    line-height: 30px;
```

Comment:
There's no need for font-weight here (or anywhere where you got 400) since it's the default one. Secondly, the preferred way to include line-height is not the fixed number like you have here as it's not responsive and sometimes can mess things up. Personally I stick to the 'line-height in pixels value divided by font-size in pixels value' kind of a method, somewhere else in the sccss you wrote it with % so my advice would be to choose either of the two and never use pixels again. Again, consistency of the code is the key :>